docs/development.md

```diff
@@ -155,6 +156,31 @@ To "log out", submit the following API call:
 POST http://localhost:8080/_ah/login?action=Log+Out
 ```
 
+## Running the Datastore Emulator
+
+The Datastore Emulator is an essential tool that we use to locally simulate production Datastore environment during development and testing of relevant features. For more information about the datastore emulator, refer to Google's [official documentation](https://cloud.google.com/datastore/docs/tools/datastore-emulator). 
+
+### Steps to local setup
+**1. Installing the Datastore Emulator**    
+After you have installed the [Google Cloud SDK](https://cloud.google.com/sdk/docs/install) which contains the `gcloud` tool, the datastore emulator can be installed by running the command:
```

Comment:
Not sure if relevant to this PR, but just wondering if we could have these run as gradle tasks? I remember devs previously would have to setup google cloud sdk for deployment but @wkurniawan07 did some magic with gradle and that requirement was removed. Now it's being brought back so just checking if alternatives have been considered

Reply:
That's the magic from App Engine plugin, not anywhere near anything done by me. Datastore emulator is however an entirely different product, so it looks like Google Cloud SDK becomes a necessity once again.